types/txn.go

```diff
@@ -431,11 +425,6 @@ func (ctx *TxParseContext) parseTransactionBody(payload []byte, pos, p0 int, slo
 
 	p = dataPos + dataLen
 
-	if txType == DepositTxType {
-		log.Debug("MMDBG erigon-lib finished parsing DepositTxType")
-		return p, nil
```

Comment:
Skipping this return changes the logic?

Reply:
I've rebased on the current boba-develop and I've restored the logic here (now only removing the log.Debug line).